packages/flutter_tools/test/logger_test.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
New hotness is to put the test in the same folder structure as the file under test, so this belongs in `test/src/base/logger_test.dart`